src/types/singleModel.js

```diff
@@ -0,0 +1,35 @@
+/* @flow */
+
+'use strict';
+
+const BaseType = require('./base');
+
+class SingleModel extends BaseType {
+
+    constructor (parentModel: Object, Model: Function, options: Object = {}) {
+        super(parentModel, options);
+
+        Object.assign(this, {
+            _ModelConstructor: Model,
+            _model: new Model(parentModel),
```

Comment:
`this._model` can still be there, you just cant construct a new model for it. This is really the same problem that I am talking about it MultiModel